src/controllers/users.controller.ts

```diff
@@ -60,6 +60,18 @@ class UsersController {
       next(error);
     }
   };
+
+  public manageGroups = async (req: Request, res: Response, next: NextFunction) => {
+    try {
+      const data = req.body;
+      const userId = Number(req.params.id);
+      const groupData = await this.userService.manageGroups(userId, data);
```

Comment:
In the way it is written, you expect an object to be received in the body of the request to manage users directly in the database and this SHOULD NEVER BE DONE, you always have to validate the properties that come in the body and, if everything is correct, we proceed to interact with the database. This code could cause a direct XSS attack on the Users database.